pattern-1/tile/tile.yml

```diff
@@ -22,19 +22,19 @@ label: WSO2 API Manager 2.6.0
 description: PCF Tile for WSO2 API Manager 2.6.0
 
 stemcell_criteria:
-  os: ubuntu-trusty
+  os: ubuntu-xenial
   requires_cpi: false
-  version: '3541'
+  version: '170.15'
 
 forms:
-- name: carbon_db
-  label: Carbon DB connection information
-  description: WSO2 API Manager - Carbon Database connection information
+- name: reg_db
+  label: Registry DB connection information
+  description: WSO2 Enterprise Integrator - Registry Database connection information
```

Comment:
The description should be **WSO2 API Manager - Registry Database connection information**